src/DetailsView/reports/components/instance-list-group-header.tsx

```diff
@@ -7,11 +7,11 @@ import { NewTabLink } from '../../../common/components/new-tab-link';
 import { NamedSFC } from '../../../common/react/named-sfc';
 import { RuleResult } from '../../../scanner/iruleresults';
 import { OutcomeChip } from './outcome-chip';
-import { OutcomeType } from './outcome-type';
+import { RequirementOutcomeType } from './outcome-type';
 
 export interface InstaceListGroupHeaderProps {
```

Comment:
Noticed a typo in an unchanged line:
```suggestion
export interface InstanceListGroupHeaderProps {
```